### ISSUE TITLE
Not throwing an error

### ISSUE THREAD

**Issue by joshuaalpuerto** [react-boilerplate/react-boilerplate#2297]
## Issue Type

- [x] Bug (https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md#bug-reports)

## Description
`console.log` not displaying error.

I added an error under `app/containers/HomePage/actions.js`
```js
/*
 * Home Actions
 * ...
 */

shouldThrowError // <---line 20
```

This should throw an error on `console.log` but it doesn't.

### React Boilerplate v3.4.0
I went to `releases` tried to download version `v3.5.0` but it gives me `3.4.0`.

Anyway, I added the same code and it does throw an error which should be.

<img width="1616" alt="screen shot 2018-07-28 at 7 08 48 pm" src="https://user-images.githubusercontent.com/7126484/43356022-5b92a914-929b-11e8-83f3-5b3176cd1150.png">

I know this can be fix by simply `eslint --fix .` but the thing is this should throw an error. It's quite hard to debug especially if it's just **white** page.

## Steps to reproduce
[https://github.com/joshuaalpuerto/rbp-repro-error-swallowed](https://github.com/joshuaalpuerto/rbp-repro-error-swallowed)

1.  Clone the repository with `git@github.com:joshuaalpuerto/rbp-repro-error-swallowed.git`
2. `yarn`
3. `yarn start`

# Versions
- React-Boilerplate 3.6.0
- Node version 8.9.0 & 10.7.0
- Npm version 5 & 6
- Browser: all browsers

**Comment by joshuaalpuerto** [comment]
It seems `react-loadable` swallowing the error. 

Importing the component directly
```js
import HomePage from 'containers/HomePage/Loadable';

//to

import HomePage from 'containers/HomePage';
```

Will throw the error on `console.log`

It seems there a [PR](https://github.com/jamiebuilds/react-loadable/pull/129/files) exist for this issue.

**Comment by joshuaalpuerto** [comment]
**[UPDATE]**

It seems already merge but the `5.4.1` hasn't been released yet by `react-loadable`. 

**SOLUTION**
Downgrade to `5.3.0` Let me know if this is something I can PR but I think `react-loadble` should release the fix.

Cheers!

**Comment by joshuaalpuerto** [comment]
Hi @julienben,

Thanks for the hard word by the way.

> indicating that there's a 5.4.1 release for react-loadable?

It's actually on the comment of the [PR](https://github.com/jamiebuilds/react-loadable/pull/129#issuecomment-399153434). Too bad it hasn't been release yet.

> I'm not seeing the bug, the same action on my side gives me a console error

I can still produce it cloning my [repro](https://github.com/joshuaalpuerto/rbp-repro-error-swallowed). It won't really give me an error and it's just loading.

**Comment by julienben** [comment]
Ah gotcha! Thanks for linking the PR, I think it can't hurt to ask the good people of react-loadable if they can do a quick release to fix this bug. Ideally it would be fixed before we release a new version here.

If it isn't though, we'll consider downgrading to 5.3.0. You can do that in your own project in the meantime of course!

And I'll reopen this since it's still a relevant issue for RBP and we want to make sure it's fixed one way or another.

**Comment by n4gashi** [comment]
Hello peeps, 
I guess the release have not been out yet because the tests were failing on the PR I've been submitting. Changes have been made since and I'm now waiting for someone from the team to get me to merge my change and then it is going to be up to them afterwards. 

Long story short, it should come soon if they decide to make a release just right after my PR has been merged :)

[PR #129 @react-loadable](https://github.com/jamiebuilds/react-loadable/pull/129/files)

**Comment by n4gashi** [comment]
Yeah 5.5 have been released with a revert of my first set of changes as it was breaking the tests. I've been releasing a new patch that does not break the tests but it's still pending for approval.

**Comment by julienben** [comment]
Update: react-loadable hasn't fixed this issue yet so I downgraded to 5.3.1. It will be part of the next release (unless they publish a fix before then) so I'm closing this for now.
